### ISSUE TITLE
Tricky PTR

### ISSUE THREAD

**Issue by szepeviktor** [kalmargabor/crawler-check#1]
Could a tricky PTR record fool crawler-check?
`host.google.com.mydomain.net`

```php
stristr($host, $validHost)
```

**Comment by szepeviktor** [comment]
I suggest these two **endings**: `.googlebot.com.` `.google.com.`

```php
    private function str_endswith($haystack, $needle)
    {

        return (strpos($haystack, $needle) + strlen($needle) === strlen($haystack));
    }
```

or https://stackoverflow.com/a/834355

**Comment by kalmargabor** [comment]
Nice catch, thanks! Fixed in 1.0.1
